src/AuthKeyValueUser.php

```diff
@@ -228,12 +225,7 @@ public function getPropertyValues($property)
             $extraParameters = $this->getUserExtraParameters();
             // the element has already been accessed
             if(!empty($extraParameters) && array_key_exists($property, $extraParameters)){
-                if(!is_array($extraParameters[$property])){
-                    $returnValue = array($extraParameters[$property]);
-                } else {
-                    $returnValue = $extraParameters[$property];
-                }
-
+                $returnValue = array($extraParameters[$property]);
```

Comment:
Are you fully sure that this change is really needed?
I mean in prev case you have got `array($extraParameters[$property])` only in case `$extraParameters[$property]` is not an array. And now you will do that in any case. So in case it's array -> it would be as `array(array(...))`

Reply:
I just changed it because on my checking,  it never goes on the else, but someone more experienced on this can prove me wrong.